### ISSUE TITLE
editor.unicodeHighlight.ambiguousCharacters not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#158381]
Have an ambiguous character in your editor. Notice how we render this specially so user can detect the ambiguous character.
Turn on a screen reader and notice that the screen reader does not ready anything specially when you visit the line with the ambiguous character. 

Some ideas on what we could do:
- play an aria live event when caret is on the ambiguous character. The aria live event would announce the exact character code.
- play an audio cue when on a line with an ambiguous character

Assigning to September so we reserve some time to think about this next milestone.

**Comment by isidorn** [comment]
Feedback from @robperez:

1. We should use the aria live event. It should play when the cursor is on the exact line / column location
2. The live announcement should be "U+202a, ambiguous character". Since there is a visual warning for the character, the screen reader user should also get a warning by reading "ambiguous character"

**Comment by rperez030** [comment]
Since vS Code already uses a special textarea for the optimized screen reader mode, I wonder if it would be possible to simply replace the ambiguous character with the new string so that it will be read when screen reader read the entire line and be reflected in Braille. I imagine that one problem with this approach is that we would have to find a way to make this string not editable by the user. Leaving the idea here in case it helps.
